css-ui-3/outline-003.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html lang=en>
+  <meta charset=utf-8>
+  <title>CSS-UI test: outline & layout</title>
+  <link rel="author" href="http://florian.rivoal.net/">
+  <meta name=flags content="">
+  <meta name=assert content="Outline does not influence size or position, and has no impact on layout.">
+  <link rel=help href="https://drafts.csswg.org/css-ui-3/#outline-props">
+  <link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<style>
+div {
+  background: red;
+  width: 100px;
+  height: 100px;
+}
+span:nth-of-type(1) {
+  float: left;
+  width: 30px;
+  height: 30px;
+  outline: solid 10px green;
+  background: green;
+
+  position: relative;
+  top: 10px;
+  left: 10px;
+}
+span:nth-of-type(2) {
+  float: left;
+  width: 0;
+  height: 0;
+  outline: solid 25px green;
```

Comment:
Because both the outlines and the boxes are green, it should not matter how they stack.
I suppose that the spec is vague enough that you could imagine that in case of overlapping boxes, the outline would be allowed to become pink with flashing semi-transparent yellow dots, pulsating width and a random offset, but that is not what it intends, and is not what people implement.

So long as what implementations do is stack or paint the outline differently, I believe this test should pass, as what is on top of what should make no difference to the final result. That said, I don't think this test is particularly important, so if you don't like it, I'm ok with removing or replacing it.